pkg/satellite/satelliteweb/api.go

```diff
@@ -68,32 +76,30 @@ func getToken(req *http.Request) string {
 }
 
 // getQuery retrieves graphql query from request
-func getQuery(req *http.Request) (query string, err error) {
+func getQuery(req *http.Request) (query queryJSON, err error) {
 	switch req.Method {
 	case http.MethodGet:
-		return req.URL.Query().Get(satelliteql.Query), nil
+		query.Query = req.URL.Query().Get(satelliteql.Query)
+		return query, nil
 	case http.MethodPost:
-		return queryPOST(req)
+		return query, queryPOST(req, &query)
 	default:
-		return "", errs.New("wrong http request type")
+		return query, errs.New("wrong http request type")
 	}
 }
 
 // queryPOST retrieves query from POST request
-func queryPOST(req *http.Request) (query string, err error) {
+func queryPOST(req *http.Request, query *queryJSON) (err error) {
 	switch typ := req.Header.Get(contentType); typ {
 	case applicationGraphql:
 		body, err := ioutil.ReadAll(req.Body)
-		return string(body), utils.CombineErrors(err, req.Body.Close())
+		query.Query = string(body)
+		return utils.CombineErrors(err, req.Body.Close())
```

Comment:
return utils.CombineErrors(err, req.Body.Close())

this line could be moved from applicationGraphql and applicationJSON cases and placed after the switch